src/presentation/http/router/note.test.ts

```diff
@@ -50,48 +53,83 @@ describe('Note API', () => {
   });
 
   describe('GET note/:notePublicId ', () => {
-    test('Returns note with access rights by public id with 200 status when note is publicly available', async () => {
-      const user = await global.db.insertUser();
+    test.each([
+      /** Returns 200 if user is team member with a Read role */
+      {
+        role: MemberRole.Read,
+        isPublic: false,
+        isAuthorized: true,
+        expectedStatusCode: 200,
+      },
 
-      const note = await global.db.insertNote({
-        creatorId: user.id,
-      });
+      /** Returns 200 if user is team member with a Write role */
+      {
+        role: MemberRole.Write,
+        isPublic: false,
+        isAuthorized: true,
+        expectedStatusCode: 200,
+      },
 
-      await global.db.insertNoteSetting({
-        noteId: note.id,
+      /** Returns 200 if note is public */
+      {
+        role: null,
         isPublic: true,
-      });
+        isAuthorized: false,
+        expectedStatusCode: 200,
+      },
 
-      const response = await global.api?.fakeRequest({
-        method: 'GET',
-        url: `/note/${note.publicId}`,
-      });
+      /** Returns 403 if user is not in the team */
+      {
+        role: null,
+        isPublic: false,
+        isAuthorized: true,
+        expectedStatusCode: 403,
+      },
 
-      expect(response?.statusCode).toBe(200);
+      /** Returns 403 if user is not authorized */
+      {
+        role: null,
+        isPublic: false,
+        isAuthorized: false,
+        expectedStatusCode: 403,
+      },
+    ])
```

Comment:
unauthorize should return 401 status code, 403 is forbidden, i think you need to check logic

Reply:
You are right, the policy notePublicOrUserInTeam can now throw 401. Fixed

Reply:
You're right. Now the policy "notePublicOrUserInTeam can throw 401"